deploy/charts/cert-manager/templates/crds.yaml

```diff
@@ -0,0 +1,154 @@
+{{- if and .Release.IsInstall .Values.crds.create -}}
```

Comment:
This would delete crds on upgrade.

Reply:
Hm... I don't think so. That install only once and not modify it after upgrade. I checked it.

Reply:
example: https://github.com/helm/charts/blob/master/stable/prometheus-operator/templates/prometheus-operator/crd-prometheus.yaml